src/rgw/rgw_op.h

```diff
@@ -104,6 +105,8 @@ RGWOp() : s(nullptr), dialect_handler(nullptr), store(nullptr),
   virtual uint32_t op_mask() { return 0; }
 
   virtual int error_handler(int err_no, string *error_content);
+
+  boost::function<void()> dump_access_control_f();
```

Comment:
@cbodley : I am thinking of removing dump_access_control_f(), since I have already reverted to calling dump_access_control() directly. Let me know what you think.